docs/rule/no-action.md

```diff
@@ -4,7 +4,7 @@
 
 What's wrong with `{{action}}`?
 
-"Action" is an overloaded term in Ember parlance. Actions are:
+"Action" is an overloaded term in Ember idiom. Actions are:
```

Comment:
Not a big deal but parlance seems a bit more fitting here?

Reply:
While somewhat unrelated to this PR, I'm in favor of this change.

Maybe a nice alternative would be `diction`, `language`, or `vocabulary`.

Or, simply drop the word all-together.

```
"Action" is an overloaded term in Ember. Actions are:
```

Reply:
Any of these new or existing options sound fine to me. Feel free to choose whatever you think will be most-easily understood.

Reply:
ok, I proposed the change because I didn't know what parlance is.  But looking at the alternatives, I also feel like we can drop it all together.  since it already says `is an overloaded term`